zapzap/controllers/WebView.py

```diff
@@ -78,6 +79,8 @@ def _configure_profile(self):
         self.profile.setHttpCacheType(
             self.QWEBENGINE_CACHE_TYPES.get(SettingsManager.get(
                 "performance/cache_type", "DiskHttpCache")))
+        
+        ExtensionManager.set_extensions(self.profile)
```

Comment:
Calls the static method that applies the extensions to the QWebEngineView profile